rightmesh/contracts/MeshToken.sol

```diff
@@ -14,19 +18,41 @@ contract MeshToken is CappedToken, PausableToken {
    */
   mapping (address => bool) public allowedTransfers;
 
+  /**
+   * @dev variable to keep track if token has been paused once in its lifetime.
+   */
+  bool public pausedOnce = false;
```

Comment:
public will create an automatic getter for this variable which may or may not be what we want. Do we expect outside contracts (ie. contracts *not* derived from this one) to contain logic that checks the value of pausedOnce?

If not, maybe we should go with:
internal -> this and derived contracts can access
private -> only this contract can access